{{ cookiecutter.__repo_name }}/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["ms-python.black-formatter", "BdSoftware.format-on-auto-save"]
```

Comment:
Is **BdSoftware** really neded? Does not `files.autoSave` actually implements this already?
I would also add **isort**

Reply:
See comment below.